### PR TITLE
CB-4-team

### PR DESCRIPTION
**Pull request by saitanjavuru**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds in the Team component

* **What is the current behavior?What does the code do before you made the new changes?** 
The code does nothing

* **What is the new behavior?**
The component displays cards with title and description of the team members.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No

* **Other information**:
(Team component after 11th commit)
Team component after 11th commit

### CODE REVIEW

**Review by AdemOdza — APPROVED** [review]
Looks good! You are right, the component can be customized later once we have a set theme.

**Review by alekhyapinnamaneni — COMMENTED** [review]
Everything looks great! The only problem I see is a merge conflict on LandingPage.js, but that will be a quick fix. This can be done later, but I would also suggest changing the colors to match the dark green and cream/white theme we decided on.

**Review by Stacey-Masi — CHANGES_REQUESTED** [review]
I know you're still working on the style and adding the theme, but I thought it'd be cool to list our class years (ex: undergrad sophomore class of 2023), link our individual GitHub/LinkedIn, or career goals/hobbies like a meet the developers type of thing.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import React from 'react';
+import { Card, Button, CardTitle, CardText, CardSubtitle, CardDeck} from 'reactstrap';
+import './Team.css'
+
+document.body.style.backgroundColor="#e3c3b2";
+
+function Team(){
+    return (
+        <div className ="container">
+        <h1 className = "home">Meet the Team!</h1>
```
Comment (sunghyun96) [inline frontend/src/components/LandingPage/Team/Team.js]: What do you think of changing this h1 className from just "home" to "home display-4" just to keep that header font consistent through the landing page?

**Review by sunghyun96 — CHANGES_REQUESTED** [review]
This looks good Sai! I left a comment for you to check out!

**Review by Stacey-Masi — APPROVED** [review]
This looks great, Sai!

File: frontend/src/components/LandingPage/Team/Team.js (hunk 1 above)
Comment (saitanjavuru) [reply frontend/src/components/LandingPage/Team/Team.js]: For sure!

File: frontend/src/components/LandingPage/Team/Team.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/*
+This component will have a title and description of the team members.
+*/
+
+import React from 'react';
+import { Card, Button, CardTitle, CardText, CardSubtitle, CardDeck} from 'reactstrap';
+import './Team.css'
+
+document.body.style.backgroundColor="#e3c3b2";
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Lets move all styling to the stylesheet. One way to change the bground color for the component is to add the color to the container div on line 13.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    
+                    <Card body className = "text-center cardStyling">
+                        <CardTitle tag="h4">Alekhya Pinnamaneni</CardTitle>
+                        <br></br>
+                        <CardSubtitle tag ="h5">Major and Class</CardSubtitle>
+                        <br></br>
+                        <CardText tag ="h6">(Info about ourselves)</CardText>
+                        <br></br>
+                        <Button href="https://github.com/alekhyapinnamaneni" target="_blank" inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>GitHub</Button>
+                        <Button inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>LinkedIn</Button>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Can we move the inline styling to the stylesheet.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+                    <Card body className = "text-center cardStyling">
+                        <CardTitle tag="h4">Sung Hyun Hwang</CardTitle>
+                        <br></br>
+                        <CardSubtitle tag ="h5">Major and Class</CardSubtitle>
+                        <br></br>
+                        <CardText tag="h6">(Info about ourselves)</CardText>
+                        <br></br>
+                        <Button href="https://github.com/sunghyun96" target="_blank" inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>GitHub</Button>
+                        <Button inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>LinkedIn</Button>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Here as well.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+                    <Card body className = "text-center cardStyling">
+                        <CardTitle tag="h4">Stacey Masi</CardTitle>
+                        <br></br>
+                        <CardSubtitle tag="h5">Major and Class</CardSubtitle>
+                        <br></br>
+                        <CardText tag="h6">(Info about ourselves)</CardText>
+                        <br></br>
+                        <Button href="https://github.com/Stacey-Masi" target="_blank" inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>GitHub</Button>
+                        <Button inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>LinkedIn</Button>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Here as well.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+                    <Card body className = "text-center cardStyling">
+                        <CardTitle tag="h4">Sai Abhiroop Tanjavuru</CardTitle>
+                        <br></br>
+                        <CardSubtitle tag="h5">Major and Class</CardSubtitle>
+                        <br></br>
+                        <CardText tag="h6">(Info about ourselves)</CardText>
+                        <br></br>
+                        <Button href="https://github.com/saitanjavuru" target="_blank" inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>GitHub</Button>
+                        <Button inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>LinkedIn</Button>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Here as well.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+                    <Card body className = "text-center cardStyling">
+                        <CardTitle tag="h4">Adem Odza</CardTitle>
+                        <br></br>
+                        <CardSubtitle tag="h5">Major and Class</CardSubtitle>
+                        <br></br>
+                        <CardText tag="h6">(Info about ourselves)</CardText>
+                        <br></br>
+                        <Button href="https://github.com/AdemOdza" target="_blank" inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>GitHub</Button>
+                        <Button inverse style={{backgroundColor: '#2D4D46', borderColor: '#F1EEE9'}}>LinkedIn</Button>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Here as well.

**Review by Codubee — CHANGES_REQUESTED** [review]
This looks great Sai! Fantastic job. Just one thing that I addressed.

**Review by sunghyun96 — APPROVED** [review]
Thanks for addressing that header font!

**Review by Codubee — APPROVED** [review]

**Pull request merged**